src/pages/signage/index.js

```diff
@@ -0,0 +1,274 @@
+    }
+  };
+  
+  const jumpToEvent = (id) => {
+    const event = events.find(ev => ev.id === id);
+    props.publishDate(event.start_date);
+  };
+  
+  const viewSign = () => {
+    window.open(`${window.SIGNAGE_BASE_URL}/${template}#/?summit=${summit.id}&location=${locationId}`);
+  };
+  
+  const pushUpdates = () => {
+    props.publishSignUpdates(template, jumpDate, staticBannerLoc);
+  };
+  
+  if(!summit.id || !locations) return(<div />);
+  
+  const eventsColumns = [
+    { columnKey: 'id', value: 'Id', sortable: true },
+    { columnKey: 'title', value: T.translate("signage.title"), sortable: true },
+    { columnKey: 'speakers_str', value: T.translate("signage.speakers")},
+    { columnKey: 'floor_loc', value: T.translate("signage.floor_loc")},
+    { columnKey: 'start_date_str', value: T.translate("signage.start_date"), sortable: true},
+    { columnKey: 'end_date_str', value: T.translate("signage.end_date")},
+  ];
+  
+  const eventsTableOptions = {
+    sortCol: props.order,
+    sortDir: props.orderDir,
+    actions: {
+      custom: [
+        {
+          name: 'jump',
+          tooltip: 'jump to this event',
+          icon: <i className="fa fa-share" />,
+          onClick: jumpToEvent,
+        }
+      ],
+    }
+  };
+  
+  return (
+    <div className="signage-wrapper">
+      <Breadcrumb data={{ title: T.translate("signage.signs"), pathname: match.url }} />
+      <div className="container">
+        <div className={styles.header}>
+          <div className="row form-group">
+            <div className="col-md-4">
+              <label> {T.translate("signage.location")} </label>
+              <LocationGroupedDropdown
+                locations={locations}
+                value={locationId}
+                placeholder={T.translate("signage.placeholders.locations")}
+                onChange={op => onChangeLocation(op.value)}
+              />
+            </div>
+            <div className="col-md-4">
+              <label> {T.translate("signage.template")} </label>
+              <Dropdown
+                placeholder={T.translate("signage.placeholders.template")}
+                value={template}
+                onChange={ev => setTemplate(ev.target.value)}
+                options={templateOptions}
+              />
+            </div>
+            <div className="col-md-4">
+              <button className="btn btn-default" onClick={viewSign}>
+                {T.translate(`signage.view_sign`)}
+              </button>
+            </div>
+          </div>
+          <div className="row form-group">
+            <div className="col-md-4">
+              <label> {T.translate("signage.date")} </label>
+              <DateTimePicker
+                format={{date:"YYYY-MM-DD", time: "HH:mm"}}
+                inputProps={{placeholder: T.translate("signage.placeholders.date")}}
+                timezone={summit.time_zone_id}
+                onChange={ev => setJumpDate(ev.target.value.unix())}
```

Comment:
this date picker should emulate the original behavior
https://github.com/fntechgit/openstacksignage/blob/39a896a54034106a378f4831e8198de65b011698/admin.html#L478